themes/default/content/blog/finops-with-pulumi/index.md

```diff
@@ -0,0 +1,126 @@
+
+The best first step in allocation of costs to get you on your FinOps Journey is typically done with consistent and enforced tagging of cloud resources.  Creating a Tag Policy as a first step is covered below.
+
+## Reactive vs. Proactive Optimization
+
+Most FinOps processes today are reactive; once you have received your bill, you will observe (and pay) the costs and resources accounted for, review your architecture and make necessary adjustments such as reserving, reallocating or removing resources.  Next month, you will do it again, also factoring in your company’s cloud growth, application design changes, team shifts and other dynamics that occured in the meantime.  Many platforms, tools, and job descriptions are oriented toward this reactive optimization process.
+
+Proactive optimization and enforcement means providing real time feedback on how cloud engineering architectural changes may impact cost and creating preventative policies which do not allow for budget-busting cloud services to be provisioned.  This is an evolving practice in many modern cloud-centric organizations.  As more cloud operations are defined as code, this preventative, proactive optimization model is becoming more popular.
+
+Financial optimization can also come in the form of Strategic Sourcing; most cloud providers offer multiple discount mechanisms, including large enterprise discount plans for committed consumption that may extend to third party SaaS software purchased through the Cloud Marketplaces, as Pulumi is.  This level of optimization will provide dramatic savings on your top-line spend.
+
+## FinOps with Pulumi
+
+Using best Cloud Engineering practices with Pulumi, you have the opportunity to implement both proactive and reactive models to control your cloud spend.
+
+As an infrastructure as code platform, Pulumi is in the unique position to be proactive about managing your costs.  You can design guardrails on your provisioning to limit expensive resource types or quantities, direct resources to reserved allocations, ensure those resources are tagged and traceable to the correct cost center, connected to your observability and dedicated FinOps platforms, and provide preventative visibility and control.  Consider uses for CrossGuard Policy-as-Code, Multilanguage Components for defining resource abstractions. and Automation API to package up orchestration into web service endpoints, as described below.
```

Comment:
Since the first instance of multi-language components occurs in this paragraph, I also suggest adding "(MLCs)" (without quotes) after it, to define the acronym on first use